tasks/__init__.py

```diff
@@ -75,53 +76,39 @@ def test_travis_on_prod(ctx):
 @task
 def test_travis_part_one(ctx):
     """Run first group of tests on the browser defined by TEST_BUILD."""
-    flake(ctx)
-
-    part_one_files = ['test_dashboard.py', 'test_institutions.py', 'test_landing.py',
-                      'test_login.py', 'test_meetings.py', 'test_my_projects.py',
-                      'test_navbar.py']
-    part_one_file_names = ['tests/{}'.format(x) for x in part_one_files]
-
-    print('Testing part one modules in "/test/" in {}'.format(os.environ['TEST_BUILD']))
-    retcode = test_module_wo_exit(ctx, params=part_one_file_names)
-
-    # retcodes: http://doc.pytest.org/en/latest/usage.html#possible-exit-codes
-    if retcode != 1:
-        sys.exit(retcode)
-
-    part_one_file_names = ['--last-failed', '--last-failed-no-failures', 'none'] + part_one_file_names
-
-    for i in range(1, 4):
-        print('Retesting part one failures, iteration {}, in "/test/" '
-              'in {}'.format(i, os.environ['TEST_BUILD']))
-        retcode = test_module_wo_exit(ctx, params=part_one_file_names)
-        if retcode != 1:
-            break
-
-    sys.exit(retcode)
+    all_test_files = glob.glob('tests/test_*.py')
+    all_test_files.sort()
+    midpoint = len(all_test_files) // 2
```

Comment:
I love the double slash division!

Reply:
If you and Amanda needs / prefers, we can add some comments on the usage for both `//` and list slicing `:`.